packages/services/src/contents/index.ts

```diff
@@ -1505,6 +1511,7 @@ export class Drive implements Contents.IDrive {
   private _apiEndpoint: string;
   private _isDisposed = false;
   private _fileChanged = new Signal<this, Contents.IChangedArgs>(this);
+  contentProviderRegistry: IContentProviderRegistry;
```

Comment:
What do you think about narrowing the API a bit more by:

```suggestion
  /**
  * Content provider registry.
   * @experimental
   */
  readonly contentProviderRegistry: IContentProviderRegistry = new ContentProviderRegistry();
```

and moving it up to be with all the other public members?